src/components/collaborative-editing/components/avatars/index.js

```diff
@@ -35,13 +36,24 @@ export function CollaborativeEditingAvatars( { peers } ) {
 
 export function CollaborativeEditingAvatar( { peer } ) {
 	const [ isVisible, setIsVisible ] = useState( false );
+	const { selectBlock } = useDispatch( blockEditorStore );
+
+	const focusPeerSelectedBlock = () => {
+		if ( peer && peer.start && peer.start.clientId ) {
+			selectBlock( peer.start.clientId );
+		}
+	};
 
 	return (
 		<div
 			className="iso-editor-collab-avatars__avatar"
 			aria-label={ peer.name }
 			onMouseEnter={ () => setIsVisible( true ) }
 			onMouseLeave={ () => setIsVisible( false ) }
+			tabIndex={ -1 }
+			role="button"
+			onKeyDown={ () => {} }
+			onClick={ focusPeerSelectedBlock }
```

Comment:
> Any a11y concerns?

Yep, here's a good article on the clickable div antipattern and why you don't want to implement buttons from scratch. TL;DR is to use a `button` tag 😄 https://benmyers.dev/blog/clickable-divs/

Reply:
Great reading, thanks!